opae-libs/cmake/config/config.h.in

```diff
@@ -231,6 +231,7 @@
 #define OPAE_ASE_CFG_INST_PATH "@CMAKE_INSTALL_PREFIX@/share/opae/ase/opae_ase.cfg"
 
 #define OPAE_MODULE_SEARCH_PATHS        \
+  "@LIBRARY_OUTPUT_PATH@/",             \
```

Comment:
We don't want this for the production case, as it violates our security standards.  Use LD_LIBRARY_PATH instead.

Reply:
I will revert it. Thanks.